OnlineFlightSearchAPITestCases/ControllerTests/SearchFlightsControllerTests.cs

```diff
@@ -19,16 +20,21 @@ public SearchFlightsControllerTests()
                 .UseStartup<Startup>());
         }
 
+        // Will Ideally return StatusCode 200 OK. Since No Data is present ValidationMessage is thrown hence the BadRequest
         [Theory]
         [InlineData("BUD", "LTN")]
-        public async Task SearchFlightDetails_ValidRequest_ReturnsHttpStatusNoContent204(string startLocation, string destination)
+        public async Task SearchFlightDetails_ValidRequest_ReturnsHttpStatusBadRequest400(string startLocation, string destination)
         {
             var client = server.CreateClient();
-            var request = $"/api/SearchFlightDetails?startLocation={startLocation}&endDestination={destination}&departureDate={DateTime.UtcNow.Date.AddDays(1).ToString("dd-MM-yyyy")}";
+            var request = TestConstants.ValidFlightSearchRequest +
```

Comment:
use string interpolation!
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated

Reply:
same for all the tests below

Reply:
Previously was using String Interpolation only. Changed it back as mentioned.